safety-quiz/covid.py

```diff
@@ -0,0 +1,26 @@
+# imports
+from flask import Flask, request, session, redirect, url_for, render_template, flash, send_from_directory, Markup, jsonify
+import sqlalchemy as sa
+from checkIn.model import User, UserLocation, Type, Training, Machine, Quiz, Question, Option, MissedQuestion, init_db, Major, College, HawkCard
+
+# Don't create new instance of app but use current one
+app = Flask(__name__)
+app.config.from_pyfile('config.cfg')
+db_session = init_db(app.config['DB'])
+app.config.update(dict(MAIL_SERVER = '10.0.8.18'),MAIL_DEFAULT_SENDER = "ideashop@iit.edu")
+
+
+@app.route('/COVID',methods=['GET', 'POST'])
```

Comment:
meant to comment here: https://github.com/iit-ideashop/safety-quiz/pull/4#discussion_r647516315

Reply:
i tested this branch and the flask app cannot access COVID route. Will need to be set up as a blueprint.